python/setup.py

```diff
@@ -21,12 +21,30 @@
 
 logger = logging.getLogger(__name__)
 
+
+class SetupSpec:
+    def __init__(self, name, description):
```

Comment:
Can you add type hints here (for the ctor) and for the instance variables?

Reply:
Make sense.